server/collectionPub/queries.js

```diff
@@ -48,10 +48,19 @@ export const getPubsInCollection = async ({ communityId, collectionId, userId })
 		);
 };
 
+const getRankInPeers = (requestedRank, ranks, moveToTop = false) => {
+	if (requestedRank) {
+		return requestedRank;
+	}
```

Comment:
Would `requestedRank` be better as the second argument in this function seeing as it's optional?

Reply:
It's not an optional argument so much as a nullable one (if only there were some way of expressing this in code!)